docs/_templates/search.html

```diff
@@ -0,0 +1,109 @@
+{% extends "layout.html" %}
+{% set title = _('Search') %}
+
+{% block body %}
+  <noscript>
+  <div id="fallback" class="admonition warning">
+    <p class="last">
+      {% trans trimmed %}Please activate JavaScript to enable the search
+      functionality.{% endtrans %}
+    </p>
+  </div>
+  </noscript>
+
+  <div id="search-results">
+  </div>
+{% endblock %}
+
+{% block footer %}
+{{ super() }}
+<script>
+
+function runSearchPageSearch(event) {
+  const urlParams = new URLSearchParams(window.location.search)
+  const query = urlParams.get('q')
+
+  const searchResultsContainer = document.getElementById('search-results')
+  addHeadingForQuery(query, searchResultsContainer)
+
+  const docSearch = docSearchReady()
+  const index = docSearch.client.initIndex('wagtail')
+  index.search(
+    query,
+    {
+      hitsPerPage: 50,
+      facetFilters: [getVersionFacetFilter()]
+    }
+  )
+  .then(({ hits }) => addResultsList(hits, query, searchResultsContainer))
+  .catch((error) => console.log(error))
+}
+
+function addHeadingForQuery(query, parentElement) {
+  const searchHeading = document.createElement('h2')
```

Comment:
```suggestion
  const searchHeading = document.createElement('h1')
```

I assume you made this a `h2` to match the current implementation but really should be `h1`.

Reply:
Yup. 👍